testing/opensmtpd-extras/APKBUILD

```diff
@@ -0,0 +1,134 @@
+# Contributor: Shiz <hi@shiz.me>
+# Maintainer: Shiz <hi@shiz.me>
+pkgname=opensmtpd-extras
+pkgver=201703132115
+pkgrel=0
+pkgdesc="OpenSMTPD addons"
+url="https://opensmtpd.org/"
+arch="all"
+license="ISC"
+depends="libevent"
+depends_filter_monkey="libressl"
+depends_queue_python="python2"
+depends_scheduler_python="python2"
+depends_table_mysql="mariadb-client-libs"
+depends_table_postgres="libpq"
+depends_table_python="python2"
+depends_table_redis="hiredis"
+depends_table_sqlite="sqlite-libs"
+makedepends="libressl-dev libevent-dev"
+makedepends_queue_python="python2-dev"
+makedepends_scheduler_python="python2-dev"
+makedepends_table_mysql="mariadb-dev"
+makedepends_table_postgres="postgresql-dev"
+makedepends_table_python="python2"
+makedepends_table_redis="hiredis-dev"
+makedepends_table_sqlite="sqlite-dev"
```

Comment:
These `makedepends_*` are quite an overkill. `makedepends` specifies dependencies that should be installed for building the abuild as whole. You can’t build just a single subackage and these dependencies are not written in APKINDEX.

Just put all build dependencies into `makedepends` and don’t overcomplicate it.